readme.md

```diff
@@ -26,21 +26,21 @@ LaTeX, which is pronounced «Lah-tech» or «Lay-tech» (to rhyme with «blech»
 
 ## Why use LaTeX?
 
-* LaTeX is free, multiplatform.
-* LaTeX is just a text document (which can be opened by any text editor), readily converted to PDF.
+* LaTeX is free and multiplatform.
+* LaTeX is just a text document (which can be opened by any text editor), which can be converted in many formats like PDF, HTML or Word Documents.

```

Comment:
*Word*-format conversion is notoriously shoddy and probably shouldn't be mentioned.